GetStream/stream-chat-android
# [2882] Remove Lottie library

### 🎯 Goal

Replace Lottie typing animation with standalone animation and remove the library.

### 🛠 Implementation details

Created a custom view with a similar animation.

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/9600921/151782839-2e65c53e-8294-465e-b41c-ae4f46c63cb9.mp4" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/9600921/151782189-6076a398-aa46-46c7-bd3f-2a7c84e4fb4d.mp4" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

1. Start typing from another client 
2. Observe typing animation in the message list header

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

_Please provide a suitable gif that describes your work on this pull request_
